subprojects/docs/docs.gradle

```diff
@@ -402,6 +404,19 @@ releaseNotesMarkdown.configure {
     group = "release notes"
 }
 
+def generateSitemap = tasks.register("generateSitemap", SitemapGenerator) {
+    description = "Generate a sitemap.xml file for generated docs"
+    group = "documentation"
+
+    source docsDir
+    dest
+    urlBase = "https://docs.gradle.org/current/"
```

Comment:
Does this make the sitemap /current/ specific? e.g., it'll break when we add/remove links in each release?

@mark-vieira is this something that would be better suited for build that publishes the docs?